graphql-dgs/src/test/kotlin/com/netflix/graphql/dgs/LocalDateTimeScalar.kt

```diff
@@ -25,19 +25,19 @@ import java.time.format.DateTimeFormatter
 
 @DgsScalar(name = "DateTime")
 class LocalDateTimeScalar : Coercing<LocalDateTime, String> {
-    override fun parseValue(input: Any?): LocalDateTime {
```

Comment:
So our kotlin users will need to make the same update? So this is a breaking change?

Reply:
Yes, those who implement _Scalars_ in Kotlin will break.